cli/applications/phonon.py

```diff
@@ -0,0 +1,155 @@
+"""
+Relax one or a list of structures using MatterSim.
+"""
+import argparse
+
+import numpy as np
+from ase import Atoms
+from ase.io import read as ase_read
+from loguru import logger
+from tqdm import tqdm
+
+from mattersim.applications.phonon import PhononWorkflow
+from mattersim.forcefield import MatterSimCalculator
+
+
+def predict_phonon(
+    atoms_list: list[Atoms],
+    find_prim: bool = False,
+    work_dir: str = None,
+    amplitude: float = 0.01,
+    supercell_matrix: np.ndarray = None,
+    qpoints_mesh: np.ndarray = None,
+    max_atoms: int = None,
+):
+    """
+    Predict phonon properties for a list of atoms.
+
+    Args:
+        atoms_list (list[Atoms]): List of ASE Atoms objects.
+        find_prim (bool, optional): If find the primitive cell and use it
+            to calculate phonon. Default to False.
+        work_dir (str, optional): workplace path to contain phonon result.
+            Defaults to data + chemical_symbols + 'phonon'
+        amplitude (float, optional): Magnitude of the finite difference to
+            displace in force constant calculation, in Angstrom. Defaults
+            to 0.01 Angstrom.
+        supercell_matrix (nd.array, optional): Supercell matrix for constr
+            -uct supercell, priority over than max_atoms. Defaults to None.
+        qpoints_mesh (nd.array, optional): Qpoint mesh for IBZ integral,
+            priority over than max_atoms. Defaults to None.
+        max_atoms (int, optional): Maximum atoms number limitation for the
+            supercell generation. If not set, will automatic generate super
+            -cell based on symmetry. Defaults to None.
+    """
+    pred_phonon_list = []
+    for atoms in tqdm(
+        atoms_list, total=len(atoms_list), desc="Predicting phonon properties"
+    ):
+        phonon = PhononWorkflow(
+            atoms=atoms,
+            find_prim=find_prim,
+            work_dir=work_dir,
+            amplitude=amplitude,
+            supercell_matrix=supercell_matrix,
+            qpoints_mesh=qpoints_mesh,
+            max_atoms=max_atoms,
+        )
+        phonon.run()
+        pred_phonon_list.append(phonon)
```

Comment:
should this return the output of `phonon.run()`?

Reply:
You are right. I have fixed it.